tools/codechecker_report_hash/README.md

```diff
@@ -0,0 +1,40 @@
+
+## Report hash generation module
+
+### Generate report hash (v1)
```

Comment:
What is the reason for naming? (v1, v2)
Shouldn't we have more descriptive names like `get_path_sensitive_hash`?